latex/delegation.tex

```diff
@@ -367,16 +409,15 @@ \subsection{Witnesses}
     \label{eq:deleg-witnesses}
     \inference[Deleg-wit]
     { \dwit{c} = (\var{vk_s}, \sigma)
-      & \var{dstate} \trans{deleg}{c} \var{dstate'}
-      & \var{cepoch} \vdash \var{pstate}
-      \trans{pool}{c} \var{pstate'}
+      & \var{slot} \vdash \var{dstate} \trans{deleg}{c} \var{dstate'}
+      & \var{slot} \vdash \var{pstate} \trans{pool}{c} \var{pstate'}
```

Comment:
this is a  more general comment about this rule style. It might not be directly obvious that it means that the rule can be executed if `pool` or `deleg` can be executed. Maybe we should state this explicitly somewhere how this is intended to be understood (I hope I didn't miss an existing description).

Reply:
it's explained in section 4, "Composition of Transition Systems" in the "Small Step Semantics for Cardano" paper.

Reply:
thanks, I haven't read this yet